pkg/cli/cmds/executions.go

```diff
@@ -124,6 +151,36 @@ func executionDelete(c *cli.Context) error {
 	return nil
 }
 
+func delete(c *cli.Context) error {
+	namespace := c.GlobalString("namespace")
+	kubeConfig := c.GlobalString("kubeconfig")
+
+	if len(c.Args()) != 1 {
+		return InvalidArgs{}
+	}
+
+	name := c.Args().First()
+
+	controllers, err := getControllers(kubeConfig, namespace)
+	if err != nil {
+		return err
+	}
+
+	_, err = controllers.executions.Get(namespace, name, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	err = controllers.executions.Delete(namespace, name, &metav1.DeleteOptions{})
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("Deleted %s\n", name)
```

Comment:
Random print

Reply:
it tells you the delete was successful when you use the cli, why is it random?